crypto/ec/ec_lcl.h

```diff
@@ -640,7 +640,7 @@ int ossl_ecdsa_sign_setup(EC_KEY *eckey, BN_CTX *ctx_in, BIGNUM **kinvp,
                           BIGNUM **rp);
 int ossl_ecdsa_sign(int type, const unsigned char *dgst, int dlen,
                     unsigned char *sig, unsigned int *siglen,
-                    const BIGNUM *kinv, const BIGNUM *r, EC_KEY *eckey);
+                    const BIGNUM *in_kinv, const BIGNUM *in_r, EC_KEY *eckey);
```

Comment:
Could we avoid this change?

Reply:
Yes certainlly, but I wonder why...
this is only a internal header file, and variable name have no impact as I think...